CHANGELOG.md

```diff
@@ -15,12 +15,13 @@ Documentation for rocRAND is available at
 
 ### Changed
 
+* Updated the default value for `-a` argument from `rmake.py` to `gfx906:xnack-,gfx1030,gfx1100,gfx1101,gfx1102,gfx1151,gfx1200,gfx1201`
 * `rocrand_discrete` for MTGP32, LFSR113 and ThreeFry generators now uses the alias method, which is faster than binary search in CDF.
 
 ## rocRAND 3.1.1 for ROCm 6.2.4
 
 ## Fixes
-
+* Fixed an issue in `rmake.py` where the list storing cmake options would contain individual characters instead of full string of option
```

Comment:
```suggestion
* Fixed an issue in `rmake.py` where the list storing cmake options would contain individual characters instead of a full string of options.
```